Build not working with provided instructions

Hello,

I have tried to follow the instructions in the README.md in a clean Ubuntu 20.04 Docker.
Here you can see my screencast: https://asciinema.org/a/2W9B9wqprV3V4nHU3G9qIgBhF

In the `make` command it fails with:

```bash
CMake Error at CMakeLists.txt:2 (project):
  No CMAKE_CXX_COMPILER could be found.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.
```

I guess that there is something that is missing in the dependency installation.
Any idea @lukicdarkoo?

Thanks!

--- a/README.md
+++ b/README.md
@@ -12,7 +12,7 @@ First, make sure the Pico SDK is properly installed and configured:
 
 ```bash
 # Install dependencies
-sudo apt install cmake gcc-arm-none-eabi doxygen libnewlib-arm-none-eabi
+sudo apt install g++ gcc-arm-none-eabi doxygen libnewlib-arm-none-eabi git python3
 git clone --recurse-submodules https://github.com/raspberrypi/pico-sdk.git $HOME/pico-sdk
 
 # Configure environment

